opentelemetry-resource-detectors/README.md

```diff
@@ -12,7 +12,8 @@ Community supported Resource detectors implementations for applications instrume
 
 ## Features
 
-| Detector       | Implemented Resources                               | Semantic Conventions       |
-| -------------- | --------------------------------------------------- | -------------------------- |
+| Detector                | Implemented Resources                               | Semantic Conventions       |
+|-------------------------| --------------------------------------------------- | -------------------------- |
 | ProcessResourceDetector | PROCESS_COMMAND_ARGS, PROCESS_PID | https://github.com/open-telemetry/opentelemetry-specification/blob/v1.6.x/specification/resource/semantic_conventions/process.md |
-| OsResourceDetector  | OS_TYPE | https://github.com/open-telemetry/opentelemetry-specification/blob/v1.6.x/specification/resource/semantic_conventions/os.md |
+| OsResourceDetector      | OS_TYPE | https://github.com/open-telemetry/opentelemetry-specification/blob/v1.6.x/specification/resource/semantic_conventions/os.md |
+| HostResourceDetector    | HOST_ID | https://github.com/open-telemetry/opentelemetry-specification/blob/v1.6.x/specification/resource/semantic_conventions/host.md |
```

Comment:
can you point to the semantic conventions repo instead? As spec repo no longer hosts semantic conventions!

Reply:
can you actually do that change everywhere in this readme!
https://github.com/open-telemetry/semantic-conventions/tree/main/docs/resource

Reply:
Fixed in last commit https://github.com/open-telemetry/opentelemetry-rust-contrib/pull/57/commits/32f6d9f4e91df713556c074ff9219fd131ca1c62